src/commands/run.rs

```diff
@@ -82,8 +82,30 @@ pub struct Args {
     pub no_color: bool,
 
     /// The report mode.
-    #[arg(short = 'm', long, default_value_t, value_name = "MODE")]
-    pub report_mode: Mode,
+    #[arg(short = 'm', long, value_name = "MODE")]
+    pub report_mode: Option<Mode>,
+}
+
+impl Args {
+    /// Applies the configuration to the arguments.
+    pub fn apply(mut self, config: crate::config::Config) -> Self {
+        self.config = match self.config {
+            Some(path) => Some(path),
+            None => {
+                if config.run.config.exists() {
+                    Some(config.run.config)
+                } else {
```

Comment:
If a user specifies a config path that doesn't exist, we should log a warning (and maybe bail?) instead of silently continuing with no config

Reply:
To avoid having an `Option` in the `RunConfig`, this can be omitted, but gets the default value of `""`. So maybe the check needs to be exists, then raise an error only if the `path` is not the empty string.